### PR TITLE
o1vm/riscv32im: add one test for addi with negative values only

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: o1vm/tests/test_riscv_elf.rs [hunk 1]
```diff
@@ -195,6 +195,7 @@ fn test_addi_negative() {
     assert_eq!(witness.registers[T0], 100);
     assert_eq!(witness.registers[T1], 50);
     assert_eq!(witness.registers[T2], (-50_i32) as u32);
```
Comment (svv232) [inline o1vm/tests/test_riscv_elf.rs:197]: i think it makes sense to add a t3 assertion here as well for -1000

File: o1vm/tests/test_riscv_elf.rs [hunk 2]
```diff
@@ -195,6 +195,7 @@ fn test_addi_negative() {
...
+    assert_eq!(witness.registers[T4], (-1500_i32) as u32);
```
Comment (svv232) [inline o1vm/tests/test_riscv_elf.rs:198]: i think it makes sense to add a t3 assertion here as well for -1000

**Review by svv232 — CHANGES_REQUESTED** [review]

File: o1vm/tests/test_riscv_elf.rs (hunk 2 above)
Comment (dannywillems) [reply o1vm/tests/test_riscv_elf.rs:199]: Addressed in 2c55e6ba2a31c62e6e6f4c2edabd2134550be340

**Review by svv232 — APPROVED** [review]
